debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerStreamingChangeEventSource.java

```diff
@@ -138,63 +136,67 @@ public void execute(ChangeEventSourceContext context) {
             LogMinerQueryResultProcessor processor = new LogMinerQueryResultProcessor(context, logMinerMetrics, transactionalBuffer,
                     dmlParser, offsetContext, schema, dispatcher, transactionalBufferMetrics, catalogName, clock);
 
-            // 2. Querying LogMiner view while running
-            Set<String> currentRedoLogFiles = getCurrentRedoLogFiles(connection, logMinerMetrics);
-            while (context.isRunning()) {
+            try (PreparedStatement fetchFromMiningView = connection
+                    .prepareStatement(SqlUtils.queryLogMinerContents(connectorConfig.getSchemaName(), jdbcConnection.username(), schema))) {
 
-                endScn = getEndScn(connection, startScn, logMinerMetrics);
-                // LOGGER.trace("startScn: {}, endScn: {}", startScn, endScn);
+                // 2. Querying LogMiner view while running
+                Set<String> currentRedoLogFiles = getCurrentRedoLogFiles(connection, logMinerMetrics);
+                while (context.isRunning()) {
 
-                metronome = Metronome.sleeper(Duration.ofMillis(logMinerMetrics.getMillisecondToSleepBetweenMiningQuery()), clock);
-                metronome.pause();
+                    endScn = getEndScn(connection, startScn, logMinerMetrics);
+                    // LOGGER.trace("startScn: {}, endScn: {}", startScn, endScn);
```

Comment:
Why is this and the next logging statement commented out?

Reply:
These lines are identical to how they were previously, I just reordered some operations above the containing `try` block.